### DISCUSSION TITLE
Consistent weekly restarts/updates.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#5664] [Suggestions]
I propose that the server be restarted at the same time on the same day every week. Preferably not during a high population time, and at a time where at least one or two staff members are available to catch if any game-breaking issues prop up from a perhaps not fully tested PR to be able to revert it.

**Comment by Fluctisonant** [comment]
Having a text channel in the Discord whenever a maintenance is about to happen/when it ends would also be really useful! 😊😊😊

**Reply by heyitsbench** [reply]
I personally think that a text channel that says "server is down for maintenance." (4:00 AM) "server is back up." (4:10 AM) week after week is a bit redundant. The two solutions (consistent updates/announced updates) are mutually exclusive in my opinion.

**Reply by 55Honey** [reply]
Thanks Fluctisonant,
I'll put this on the Agenda.

**Reply by 55Honey** [reply]
I don't think at all that scheduled updates and announcements exclude each other. It seems like an easy and welcome addition.

**Reply by heyitsbench** [reply]
> I don't think at all that scheduled updates and announcements exclude each other.

I didn't say scheduled, I said consistent. If the restart happens every week at the same time (as the suggestion...suggests), then it wouldn't make much sense (IMO) to add a channel that announces it every single time.

**Reply by 55Honey** [reply]
I'm sorry that you did not like my wording and that there was a misunderstanding in the intention.

*Consistent* updates aren't hurt at all by an announcement on discord.

**Comment by 55Honey** [comment]
Hi Bench,
Thanks for your suggestion.
As you already know, we run this as a hobby. And since we definitely aim to have staff available during updates, it's not possible to schedule them at a weekly pace.

While we would love to set a time and day and perform the maintenance then every time, people have jobs and obligations IRL and this needs to be respected.

**Comment by Nyeriah** [comment]
I’d just like to add that we already have a set update day and time, and that is Wednesday morning (EU time). 

However, since it depends on staff availability as Honey mentioned, it’s not possible to follow the plan through every week.
